docs/design.md

```diff
@@ -6,7 +6,7 @@ The choices you make when creating a connector can include which superclass and
 
 ## Choose a connection class
 
-The <span style="font-family: courier new">class</span> attribute is a unique key for your connector. Make it something that is unlikely to be used by another connector. When Tableau loads the connectors at startup, if the class is already registered, the connector will not be loaded. The class is also stamped in Tableau Workbook (.twb or .twbx) files and Tableau Datasource (.tds) files to identify what connector that particular connection was using.
+The <span style="font-family: courier new">class</span> attribute is a unique key for your connector. Make it something that is unlikely to be used by another connector. When Tableau loads the connectors at startup, if the class is already registered, the connector will not be loaded. The class is also stamped in Tableau Workbook (.twb or .twbx) files and Tableau data source (.tds) files to identify what connector that particular connection was using.
```

Comment:
Seems like we capitalize Data Source when referring the file: https://help.tableau.com/current/pro/desktop/en-us/export_connection.htm

Reply:
Tableau Data Source, since we're referring to the file.  The other changes to data source when its a generic term for the underlying database are fine.